awx/main/migrations/0139_isolated_removal.py

```diff
@@ -3,13 +3,25 @@
 from django.db import migrations, models
 
 
+def remove_iso_instances(apps, schema_editor):
+    Instance = apps.get_model('main', 'Instance')
+    Instance.objects.filter(rampart_groups__controller__isnull=False).delete()
```

Comment:
You already do this in a later migration

https://github.com/ansible/awx/blob/93e90228a2576a84c6f09e71e9b35eaabddcc7c6/awx/main/migrations/0141_remove_isolated_instances.py#L8

But I'm fine to add this, if it's to make extra sure they're gone.

Reply:
I'd rather make extra-sure than to have to make the later one a no-op.